helm/cluster-api/values.yaml

```diff
@@ -1,7 +1,7 @@
 name: cluster-api
-registry:
-  domain: quay.io
+
 images:
+  domain: quay.io
```

Comment:
I believe we are supposed to use Docker as default , look at the default on the config repo https://github.com/giantswarm/config/blob/main/default/config.yaml#L5

Reply:
Really? I thought quay was the default because of the Dockerhub ratelimiting

Reply:
No, we decided to move from Quay to Dockerhub because we wanted to use the mirroring (only works when using Dockerhub apparently).

Reply:
@whites11 @alex-dabija Are you able to weigh in here? If we decided to use Dockerhub by default why did we remove the pull secrets?

Reply:
> @whites11 @alex-dabija Are you able to weigh in here? If we decided to use Dockerhub by default why did we remove the pull secrets?

I actually don't know what the default should be.

Reply:
Which team owned the docker registry before all the reorganizations? Was it biscuit?

Reply:
I think so. And i think the reason why we use it as default is because we have mirrors set for it (but not in China)